app/containers/Vaults/index.js

```diff
@@ -487,41 +487,39 @@ const VaultsWrapper = (props) => {
     if (vault.pureEthereum) {
       vaultKey = `${vault.address}-eth`;
     }
-    let banner = null;
-    let decimals = 18;
-    if (vault && vault.token && vault.token.decimals > 0) {
-      decimals = setDecimals(vault.token.decimals);
-    }
-
-    if (
-      vault &&
-      vault.balanceOf &&
-      vault.balanceOf[0].value >= 10 ** decimals &&
-      isMigrated &&
-      !hasBeenShown
-    ) {
-      banner = <MigrationBannerSvg vaults={migratedVaultsWithBalance} />;
-      hasBeenShown = true;
-    }
     return (
-      <>
-        {banner}
-        <Vault
-          vault={vault}
-          key={vaultKey}
-          accordionKey={vaultKey}
-          active={currentEventKey === vaultKey}
-          showDevVaults={showDevVaults}
-          web3={web3}
-        />
-      </>
+      <Vault
+        vault={vault}
+        key={vaultKey}
+        accordionKey={vaultKey}
+        active={currentEventKey === vaultKey}
+        showDevVaults={showDevVaults}
+        web3={web3}
+      />
     );
   };
 
+  const showBanner = _.find(
+    migratedVaultsWithBalance,
+    (vault) =>
+      vault &&
+      vault.balanceOf &&
+      vault.balanceOf[0].value >= 10 ** setDecimals(vault.token.decimals),
```

Comment:
`setDecimals` is important here. It's used to make sure we don't show users vault migrations with dust.

Reply:
I think this _.find() is not needed since this logic is the same as the logic in line 476, if true the vault is pushed to migratedVaultsWithBalance. Should be enough to ask if showBanner = !!migratedVaultsWithBalance.length here